pynws/simple_nws.py

```diff
@@ -79,7 +84,7 @@ def parse_icon(icon):
     weather = [i.split("?")[0] for i in icon_list[6:]]
     code = [w.split(",")[0] for w in weather]
     chance = [int(w.split(",")[1]) if len(w.split(",")) == 2 else None for w in weather]
-    return time, tuple(zip(code, chance))
+    return time, list(zip(code, chance))
```

Comment:
Is this change to list intended?  I'm surprised this passes the type checking now.  I'm not sure if we should change this, but if we do, the example in the docstring would have to be fixed too.

Reply:
Seems fine to me.  Thanks!

Reply:
Unless you have something else, I'm done making changes to this PR. If you don't mind, please combine my zillion commits into a single commit when you merge this PR.